docs/contributtion/experimental-code.md

```diff
@@ -0,0 +1,167 @@
+# Experimental Code
+
+The team occasionally will author code, or accept contributions, that is
+considered experimental or unstable. The goal for this code is to ship it as
+unstable for sponsor product to leverage. During this time, the team can get
+feedback around what is working and what does not work so that changes can be
+made before an official release.
+
+Experimental code is made available in the following ways:
+
+- Components and exports prefixed with `UNSTABLE_`
+- Feature flags
+- Expanding existing APIs
+
+In all cases, experimental code should be treated as unstable and can change
+between release versions for the package that it is being imported from.
```

Comment:
maybe somehow mention "between non-only-major versions"?

Reply:
E.g. will be `minor` or `patch` enough?